src/app/(pages)/feedback/page.jsx

```diff
@@ -1,47 +1,72 @@
-        <div className="boxes">
-          <div className="options">
-            <div className="option-1">
+    <div className={styles.boxContent}>
+      <h1 className={styles.Content}> Your feedback matter to us</h1>
+      <div className={styles.boxContainer}>
+        <div className={styles.content1}>How was the consultation?</div>
+        <div className={styles.boxes}>
+          <div className={styles.boxSelection}>
+            <div className={styles.option1}>
               <h4>Loved it!</h4>
-              <input type="radio" name="" id="circle" className="circle" />
+              <input
+                type="radio"
+                name=""
+                id="circle"
+                className={styles.circle}
+              />
             </div>
-            <div className="option-2">
+            <div className={styles.option2}>
               <h4> Great</h4>
-              <input type="radio" name="" id="circle" className="circle" />
+              <input
+                type="radio"
+                name=""
+                id="circle"
+                className={styles.circle}
+              />
             </div>
-            <div className="option-3">
+            <div className={styles.option3}>
               <h4>Neutral</h4>
-              <input type="radio" name="" id="circle" className="circle" />
+              <input
+                type="radio"
+                name=""
+                id="circle"
+                className={styles.circle}
+              />
             </div>
-            <div className="option-4">
+            <div className={styles.option4}>
               <h4>Disappointing</h4>
-              <input type="radio" name="" id="circle" className="circle" />
+              <input
+                type="radio"
+                name=""
+                id="circle"
+                className={styles.circle}
+              />
             </div>
-            <div className="option-5">
+            <div className={styles.option5}>
               <h4>Terrible</h4>
-              <input type="radio" name="" id="circle" className="circle" />
+              <input
+                type="radio"
+                name=""
+                id="circle"
+                className={styles.circle}
+              />
             </div>
           </div>
-          <div className="feedback-box">
-            <h2 className="share-box">Share Your Experience!</h2>
+          <div className={styles.feedbackBox}>
+            <h2 className={styles.shareBox}>Share Your Experience!</h2>
             <textarea
               name="Text"
               placeholder="Text"
               id="text-content"
               cols="70"
               rows="15"
-              className="text-content"
+              className={styles.textContent}
             ></textarea>
-            <div className="submit-btn">Submit</div>
+            <div className={styles.submitBtn}>Submit</div>
```

Comment:
use a button instead of duv

Reply:
Change the placeholder in the textarea to something like "Leave your valuable feedback here"